email_sender/ErrorProcessor.py

```diff
@@ -0,0 +1,33 @@
+import RegExUtil as regexutil
```

Comment:
This file we can remove since suppression we are not doing as part of lambda error handling

Reply:
Removed

Reply:
This file is again appearing in the change list